### ISSUE TITLE
shouldn't this be using the logits before the last activation?

### ISSUE THREAD

**Issue by mosheliv** [qubvel/tta_wrapper#8]
otherwise we average classes and not probabilities?

**Comment by mosheliv** [comment]
sorry, it is using logits. Thanks for the great work.
